src/components/RichTextEditor/Plugins/MentionPlugin/types.ts

```diff
@@ -9,11 +9,14 @@ export enum MentionableCategory {
     USER = 'user',
 }
 
-export interface MentionComboboxItem<T extends Data = MentionableCategory> extends TComboboxItemBase {
-    category: T;
+export interface MentionItemData {
```

Comment:
This can now be a type instead of an interface as type is preferred

Reply:
updated